src/sentry/templates/sentry/emails/group_header.html

```diff
@@ -4,6 +4,11 @@
 {% url 'sentry-stream' group.organization.slug group.project.slug as old_project_link %}
 {% url 'sentry-organization-issue-list' group.organization.slug as new_stream_link %}
 
+<style type="text/css">
+  .group-header {
+    border-top: 2px solid #E7EBEE !important;
+  }
+</style>
```

Comment:
can we put these with the other .group-header styles in `src/sentry/templates/sentry/emails/email-styles.html` or is this specific to this template.

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/emails/email-styles.html#L473-L474